app/src/main/java/com/contact/yen/calculator/MainActivity.java

```diff
@@ -22,124 +45,209 @@ protected void onCreate(Bundle savedInstanceState) {
         setEvenClick();
     }
     public void init(){
-        tvInput = findViewById(R.id.tv_input);
-        tvResult = findViewById(R.id.tv_result);
-
-        btnAC = findViewById(R.id.btn_ac);
-        btnRate = findViewById(R.id.btn_phantram);
-        btnDiv = findViewById(R.id.btn_chia);
-        btn = findViewById(R.id.btn_amduong);
-        btnDot = findViewById(R.id.btn_cham);
-        btnAdd = findViewById(R.id.btn_cong);
-        btnSub = findViewById(R.id.btn_tru);
-        btnEqual = findViewById(R.id.btn_bang);
-        btnMul = findViewById(R.id.btn_nhan);
-
-        btn7 = findViewById(R.id.btn_7);
-        btn8 = findViewById(R.id.btn_8);
-        btn9 = findViewById(R.id.btn_9);
-        btn6 = findViewById(R.id.btn_6);
-        btn5 = findViewById(R.id.btn_5);
-        btn4 = findViewById(R.id.btn_4);
-        btn3 = findViewById(R.id.btn_3);
-        btn2 = findViewById(R.id.btn_2);
-        btn1 = findViewById(R.id.btn_1);
-        btn0 = findViewById(R.id.btn_0);
+        mTvInput = findViewById(R.id.tv_input);
+        mTvResoult = findViewById(R.id.tv_result);
+
+        mBtnAC = findViewById(R.id.btn_ac);
+        mBtnRate = findViewById(R.id.btn_phantram);
+        mBtnDiv = findViewById(R.id.btn_chia);
+        mBtnSoAM = findViewById(R.id.btn_amduong);
+        mBtnDot= findViewById(R.id.btn_cham);
+        mBtnAdd = findViewById(R.id.btn_cong);
+        mBtnSub = findViewById(R.id.btn_tru);
+        mBtnResult = findViewById(R.id.btn_bang);
+        mBtnMul = findViewById(R.id.btn_nhan);
+
+        mBtnNumber7 = findViewById(R.id.btn_7);
+        mBtnNumber8= findViewById(R.id.btn_8);
+        mBtnNumber9= findViewById(R.id.btn_9);
```

Comment:
mấy cái biến này cậu không dùng thì không nên khai báo biến cho rối, nên để là findviewbyid(r.id.btn).setOnClick(this)